### PR TITLE
A keras backend wrapper

### PR DESCRIPTION
**Pull request by zhichao-li**
## What changes were proposed in this pull request?
The idea is to provide some syntax sugar to  load/train/inference a keras model with bigdl 

## How was this patch tested?
unittest

### CODE REVIEW

**Comment by hkvision** [comment]
LGTM.
I will add more loss mapping (including loss function as parameter) afterwards.

**Comment by zhichao-li** [comment]
@jason-dai  Shall we merge this once the Jenkins pass?  then we can enrich the loss function or the other testing base on that. 
The extra line still needed and haven't found a solution to get rid of it.  If take a step back, we also need some logic to hack the original `Model`.

**Comment by jason-dai** [comment]
LGTM
